portfolio/src/main/webapp/family.html

```diff
@@ -5,14 +5,11 @@
     <title>My Family</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+     <script src="https://code.jquery.com/jquery-3.5.0.js"></script>
```

Comment:
Leveraging Google CDN would make the pages faster:

https://developers.google.com/speed/libraries